explorer/main.cpp

```diff
@@ -87,11 +121,6 @@ auto ExplorerMain(int argc, char** argv, void* static_for_main_addr,
   if (result.ok()) {
     // Print the return code to stdout.
     llvm::outs() << "result: " << *result << "\n";
-
-    // When there's a dedicated trace file, print the return code to it too.
-    if (scoped_trace_stream) {
-      trace_stream << "result: " << *result << "\n";
-    }
```

Comment:
This was a deliberate feature, per https://github.com/carbon-language/carbon-lang/pull/1406, and I'd prefer it remain.

Reply:
Okay, I relocated it to line 89 of the `exec_program.cpp` file and changed its name to **interpreter result**. 
I have a little doubt, should it be a part of execution phase or it should be displayed for every phase?

Reply:
I'll leave this up to @zygoloid since he's doing most of the review; the only real requirement I have is that it must be available for compiler explorer, and I don't see the phase question as affecting that.

Reply:
With this patch, it will still be in the compiler explorer trace output, unless the execution tracing is turned off (in which case it seems logical to me that the execution result would not be traced). So I think your requirement is still satisfied.